01-front-end-fundamentals/git-ssh/readme.md

```diff
@@ -0,0 +1,161 @@
+# Generating SSH Keys and Linking with Github
+
+## Learning Objectives
+* Explain how SSH serves as extra layer of security
+* Define and Differentiate between HTTPS and SSH
+* List the advantages of an SSH connection
+
+## Opening
+Secure Shell (SSH) is a command interface and protocol for securely getting access to a remote computer.
+SSH uses public-key cryptography to authenticate the remote computer and allow it to authenticate the user, if necessary. Both ends of the client/server connection are authenticated using a digital certificate, and passwords are protected by being encrypted.
+
```

Comment:
Would you recommend a doc dive? I'm not sure going to in-depth into the protocols this early would make it clear why we are doing this.